### DISCUSSION TITLE
How to edit the password on the config portal?

### DISCUSSION THREAD

**Discussion by jonogan** [khoih-prog/WiFiManager_RTL8720#8] [Q&A] [answered]
I saw in the documents that the config portal password should be configurable, but I don't see in the examples where that value can be set. Can someone point me in the right direction?

**Comment by khoih-prog** [comment, marked as answer]
Use the following function in your code

https://github.com/khoih-prog/WiFiManager_RTL8720/blob/f5b7acbe589b30c1d902f166a83b0e68a7b406aa/src/WiFiManager_RTL8720.h#L663-L667

before

https://github.com/khoih-prog/WiFiManager_RTL8720/blob/f5b7acbe589b30c1d902f166a83b0e68a7b406aa/examples/RTL8720_WiFi/RTL8720_WiFi.ino#L113
